assets/js/utils/rtcUtils.ts

```diff
@@ -134,13 +150,23 @@ export const connectWebrtc = async (
       onJoinError: () => {
         onError("Error while joining WebRTC connection");
       },
-      onTrackReady: ({ track, peer }) => {
-        if (track != null)
-          addOrReplaceTrack(peer.metadata, track, playerCallbacks[peer.metadata.email]);
-      },
       onTrackAdded: () => {
         // do nothing
       },
+      onTrackReady: ({ track, peer }: TrackContext) => {
+        if (track != null) {
+          addOrReplaceTrack(peer.metadata, track, playerCallbacks[peer.metadata.email]);
+          updateMicAndCamStatusCallback();
+        }
+      },
+      onTrackUpdated({ track, peer, metadata }: TrackContext) {
+        if (track != null) {
+          track.enabled = metadata.enabled;
+          addOrReplaceTrack(peer.metadata, track, playerCallbacks[peer.metadata.email]);
```

Comment:
Why do we need these two lines?

Reply:
We need to change the track `enabled` property to the one received in the metadata. The second line was meant to sync the track with the rest of the tracks.
 I'll check whether or not it works properly  without the second line

Reply:
I've checked it. The second line is not necessary. I'll delete that. 🧹